includes/import.form.inc

```diff
@@ -1143,14 +1177,26 @@ function islandora_multi_importer_form_submit($form, &$form_state) {
     }
   }
 
-  // Handle the ingest buttom.
+  // Handle the ingest button.
   if ($form_state['triggering_element']['#value'] == t('Ingest')) {
+    // Save all of the $form_state['input']... #options values for all 'select' type inputs
+    islandora_multi_importer_form_save_selections($form_state, $form);
```

Comment:
Ok, maybe not the ideal to pass the whole $form_state and the leave the logic to the save() function. Every time we actually change or modify the form we will need to remake the save() function. So, what about only saving/restoring what we actually pass (array) to the preprocessor. Sounds reasonable?